include/pstl_offload/internal/usm_memory_replacement.h

```diff
@@ -281,6 +302,32 @@ inline void* __attribute__((always_inline)) __libc_realloc(void *__ptr, std::siz
     return realloc(__ptr, __size);
 }
 
+#elif _WIN64
+
+inline void* __attribute__((always_inline)) _aligned_malloc(std::size_t __size, std::size_t __alignment)
+{
+    // _aligned_malloc should reject zero or not power of two alignments
+    if (!::__pstl_offload::__is_power_of_two(__alignment))
+    {
+        errno = EINVAL;
```

Comment:
I guess this is a Windows standard library contract, right?

Reply:
Yes.